app/models/vim_performance_analysis.rb

```diff
@@ -110,127 +110,6 @@ def storages_for_compute_target(target)
       end
     end
 
-    # This method answers C&U Planning question 2
```

Comment:
@Fryguy @gtanzillo can you guys look at deleted methods here, is it safe to delete these methods.

Reply:
@h-kataria I checked and is should be safe to remove these 2 methods 👍